src/Message/CoverUserUploadMessage.php

```diff
@@ -110,7 +110,7 @@ public function getVendorId()
      *
      * @return static
      */
-    public function setVendorId($vendorId): self
+    public function setVendorId(mixed $vendorId): self
```

Comment:
Mixed? Why mixed? Will it not always be an int from the database?

Reply:
Just added type to match the `@param` annotation. Didn’t investigate further.

Reply:
Could potentially be both `string` and `int`

Reply:
Have checked the code. Should only be `int`. Have changed type accordingly.